<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="configuration_tab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../../../../../../../../.designer/backup/AppIcons/icons8-gear-64.png</normaloff>../../../../../../../../../../../.designer/backup/AppIcons/icons8-gear-64.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Buoy configuration</string>
       </attribute>
       <widget class="QPushButton" name="saveConfig_pushButton">
        <property name="geometry">
         <rect>
          <x>583</x>
          <y>480</y>
          <width>200</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Save configuration</string>
        </property>
       </widget>
       <widget class="QPushButton" name="validateConfig_pushButton">
        <property name="geometry">
         <rect>
          <x>169</x>
          <y>480</y>
          <width>200</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Validate configuration</string>
        </property>
       </widget>
       <widget class="QPushButton" name="transferConfig_pushButton">
        <property name="geometry">
         <rect>
          <x>376</x>
          <y>480</y>
          <width>200</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Transfer configuration to SD card</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadConfig_pushButton">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>480</y>
          <width>114</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Load configuration</string>
        </property>
       </widget>
       <widget class="QLabel" name="gnss_settings_label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>351</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GNSS settings</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="updateRate_spinBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>350</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="updateRate_label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>350</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Update rate (Hz)</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>1048</width>
          <height>186</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="connectivity_VLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="connectivity_label">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Connectivity</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="connectivity_HLayout">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="wifi_VLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="wifi_TITLE_label">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Wifi settings</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="wifi_SSID_HLayout">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <widget class="QLabel" name="wifi_SSID_label">
                 <property name="text">
                  <string>SSID:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="wifi_SSID_lineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="wifi_PSK_HLayout">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <widget class="QLabel" name="wifi_PSK_label">
                 <property name="text">
                  <string>Password:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="wifi_PSK_lineEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="wifi_ConfirmPSK_label">
                 <property name="text">
                  <string>Confirm Password:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="wifi_ConfirmPSK_lineEdit">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="accessPoint_VLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="accessPoint_TITLE_label">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Access point settings</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="accessPoint_SSID_HLayout">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <widget class="QLabel" name="accessPoint_SSID_label">
                 <property name="text">
                  <string>SSID:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="accessPoint_SSID_lineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="accessPoint_PWD_HLayout">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <widget class="QLabel" name="accessPoint_PWD_label">
                 <property name="text">
                  <string>Password:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="accessPoint_PWD_lineEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="accessPoint_ConfirmPWD_label">
                 <property name="text">
                  <string>Confirm Password:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="accessPoint_ConfirmPWD_lineEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="data_raw_GNSS">
       <attribute name="icon">
        <iconset>
         <normaloff>../../../../../../../../../../../.designer/backup/AppIcons/icons8-cosinus-40.png</normaloff>../../../../../../../../../../../.designer/backup/AppIcons/icons8-cosinus-40.png</iconset>
       </attribute>
       <attribute name="title">
        <string>GNSS data</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>451</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_data_dop"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>451</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_data_snr"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>40</y>
          <width>541</width>
          <height>561</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_data_pos"/>
       </widget>
       <widget class="QPushButton" name="show_live_data_GNSS_pushButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>161</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Show live data</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="data_raw_METEO">
       <attribute name="title">
        <string>Meteo data</string>
       </attribute>
       <widget class="QPushButton" name="show_live_data_METEO_pushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Show live data</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>1151</width>
          <height>641</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_data_humidity"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_data_temperature"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_data_pressure"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
